plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -575,20 +611,18 @@ class NewCreateServiceModal extends Component {
         {
           className: 'button-primary flush-vertical',
           clickHandler: this.handleServiceReview,
-          disabled: errors.length !== 0,
+          disabled: false,
```

Comment:
What if we won't pass any value here? Shall we maybe refactor `FullScreenModalHeaderActions` so that we don't need to pass `disabled: false`